### PR TITLE
Changed the CSS of the webpage

### PR DESCRIPTION
**Pull request by DefCon-007**

### CODE REVIEW

**Comment by raivivek** [comment]
@DefCon-007 Use the deploy script. :P See if it works.

**Comment by raivivek** [comment]
@DefCon-007 Could you also make it a one column middle-of-the-page style? ^_^

File: feed.tmpl [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+           <div class="about-container">
+             <div class="about-container-left">
+                 {% if post['pic'] %}
+                    <img class="img-responsive" src="{{ post['pic'] }}">
+                 {% endif %}
+             </div>
+              <div class="about-container-right">
+                <h2 class="about-title"> {{post['source'] }} </h2>
+                <p class="about-description"> <b>Posted on: {{ post['created_time'] }}</b></p>
+                <p class="about-description" > {{ post['message'] }} {{post['link']}} </p>
```
Comment (raivivek) [inline feed.tmpl:21]: I don't think `post['link']` exists yet. The code for url is still there in my open PR. Feel free to review and merge it, and then take care of this. :)

**Comment by DefCon-007** [comment]
@athityakumar  Can you please explain me how to do it

File: feed.tmpl (hunk 1 above)
Comment (DefCon-007) [inline feed.tmpl:21]: Actually I wrote the code for it in the fbscrapper.py but was not able to test it due to the following error :
Traceback (most recent call last):
  File "frontend.py", line 2, in <module>
    from django.utils.encoding import smart_str, smart_unicode
ImportError: cannot import name 'smart_unicode'

Comment (raivivek) [inline feed.tmpl:21]: Just delete `smart_unicode`. It's not needed. One instance of `smart_str` when returning html from `get_html()` is sufficient.

**Comment by raivivek** [comment]
@DefCon-007 We have a deploy script in the directory - `deploy.sh`. Currently, the two lines which run the scripts are commented out.

Just run the script if you want to deploy the changes to `gh-pages`. If you also want to run the scripts, uncomment those lines and run the script. Try and let me know if it works.

**Comment by DefCon-007** [comment]
@vivekiitkgp Since While running fbscrapper.py I get : 
Traceback (most recent call last):
  File "fbscraper.py", line 16, in <module>
    with open('./ACCESS_TOKEN', 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: './ACCESS_TOKEN'

So I just ran frontend.py and after that the deploy.sh put got error : http://pastebin.com/rb0Fvtbn

**Comment by athityakumar** [comment]
For `fbscrapper.py` file to work, you need to create your own `ACCESS_TOKEN` file for FacePy API. I guess the `README.md` has info on how to generate this file.

**Comment by athityakumar** [comment]
@DefCon-007 - It wasn't in the README, but it was commented in the fbscrapper.py file.
View tutorial for creating ACCESS _TOKEN file here : http://stackoverflow.com/a/16054555/1780891

File: feed.tmpl (hunk 1 above)
Comment (raivivek) [inline feed.tmpl:21]: Fixed in 85636d.

**Comment by raivivek** [comment]
Please move all further discussion to #13.

**Pull request merged**